sqlstore/src/main/resources/db/migration/V8800__Samples_Recieved_Workflow.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE StepPositiveDouble (
+  workflowProgressId BIGINT(20) NOT NULL,
+  stepNumber         BIGINT(20) NOT NULL,
+  `input`             FLOAT unsigned NOT NULL,
+  PRIMARY KEY (workflowProgressId, stepNumber),
+  FOREIGN KEY (workflowProgressId, stepNumber) REFERENCES WorkflowProgressStep (workflowProgressId, stepNumber)
+)
+  ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+CREATE TABLE StepPositiveInteger (
+  workflowProgressId BIGINT(20) NOT NULL,
+  stepNumber         BIGINT(20) NOT NULL,
+  `input`             INT unsigned NOT NULL,
+  PRIMARY KEY (workflowProgressId, stepNumber),
+  FOREIGN KEY (workflowProgressId, stepNumber) REFERENCES WorkflowProgressStep (workflowProgressId, stepNumber)
+)
+  ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+CREATE TABLE StepSampleStock (
```

Comment:
What uses this table? (It doesn't seem to be referenced anywhere else in this PR)